src/core/dbus-manager.c

```diff
@@ -1456,7 +1456,7 @@ static int method_kexec(sd_bus_message *message, void *userdata, sd_bus_error *e
         assert(message);
         assert(m);
 
-        r = mac_selinux_access_check(message, "reboot", error);
+        r = mac_selinux_access_check(message, "kexec", error);
```

Comment:
How does this work wrt. to backwards compatiblity?

Reply:
This would break backward compatibility indeed.

As `kexec` would be a new service class permission, without an SELinux policy update, it depends on the policy's `deny_unknown` setting whether the access is allowed or not.

As this proposal adds several new service class permissions, like { preset mask revert }, one probably need to update the SELinux policy anyway.

Reply:
the approach preserves backwards-compatibility